deploy/operator/README.md

```diff
@@ -103,3 +103,28 @@ export ASSISTED_PULLSECRET_NAME=assisted-pull-secret
 export ASSISTED_PRIVATEKEY_NAME=assisted-ssh-private-key
 export SPOKE_CONTROLPLANE_AGENTS=1  # currently only single-node is supported
 ```
+
+## Running None Platform ZTP Flow (Testing only)
+
+To test None Platform with ZTP flow on top of libvirt, the following variables must be set:
```

Comment:
Could you expand on why these settings are needed? This would be helpful for newcomers to learn more about the architecture.

Also, this is specific to deploy None platform spokes, correct?

Reply:
Actually it is a way to run none platform with kube api.
When running none platform, user managed networking is set.  Therefore, we need to set it here too.
None platform doesn't have VIPs. Instead, there is load balancer (usually external), that is a central point implementing api and ingress endpoints.  The load balancer distributes the data to the relevant masters and servers. 
For this a load balancer has to be set, and the relevant dns-names must be set in libvirt.

It is specific to deploy None platform spokes.

Reply:
@ori-amizur thanks! could you add that explanation to the docs? Otherwise, someone with the same question may not find the answer :pray:

Reply:
It is part of the README in this PR. Anything else needed ?

Reply:
> It is part of the README in this PR. Anything else needed ?

I was thinking of having it in this document, not just in the PR description. The info you put in the description is important enough to have it in the documentation. IMHO.

Reply:
OK

Reply:
Added it to the README as well